plugins/action/synchronize.py

```diff
@@ -66,7 +66,15 @@ def _format_rsync_rsh_target(self, host, path, user):
             return path
 
         # If using docker or buildah, do not add user information
-        if self._remote_transport not in ['docker', 'community.general.docker', 'community.docker.docker', 'buildah', 'containers.podman.buildah'] and user:
+        if self._remote_transport not in [
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.

Reply:
done, @Akasurde , thanks for review

Reply:
Thanks.